src/eko/evolution_operator/grid.py

```diff
@@ -106,6 +106,8 @@ def __init__(
         if order == (1, 0) and method != "iterate-exact":
             logger.warning("Evolution: In LO we use the exact solution always!")
 
+        logger.info(("P" if configs.polarized else "Unp") + "olarized evolution")
```

Comment:
Let's start making logs machine parsable:
```suggestion
        logger.info(dict(polarized=configs.polarized))
```

Reply:
Maybe more correctly "Machine friendly": 
- the new proposal can be parsed also with just an `eval`, 
- syntax highlighted, and
- it is easier to search for `polarized` without having to remember to do it case insensitively